### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 6

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#332] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152324500-7f7dfe6a-dca5-4487-8a6b-5a73fb2c57cb.png)

**Comment by danieldinella** [comment]
(Procedo con una dimostrazione diversa da quella suggerita)
Consideriamo i due casi : A = 1, A = 0.
Nel caso in cui A sia uguale a 1 avremo 1 ∧ ( 1 ∨ B) == 1 , dunque 1 == 1.
Nel caso in cui A sia uguale a 0 avremo 0 ∧ ( 0 ∨ B) == 0 , dunque 0 == 0.

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217806002-f7404919-52ad-4933-a08a-e5a79a4a16ce.png)
